controller:
  manager:
    ros__parameters:
      debug: false
      use_sim_time: true
      loop_rate: 10.0
      target_speed: 5.0
      base_link_frame: "base_link"
      map_frame: "map"
      min_distance: 10.0
lifecycle_manager_controller:
  ros__parameters:
    autostart: true
    use_sim_time: true
    node_names: ["/controller/manager"]
